ingest/validation/validate_metadata.py

```diff
@@ -400,7 +762,8 @@ def cast_metadata_type(metadatum, value, id_for_error_detail, convention, metada
                     if 'ontology' in convention['properties'][metadatum]:
                         element = regularize_ontology_id(element)
                 except KeyError:
-                    # unconventional metadata will trigger this exception
+                    # study-specific metadata (ie. non-metadata convention metadata)
```

Comment:
"Non-metadata convention metadata" is pretty wordy.  "Unconventional metadata" seems simpler and sufficiently clear.

If you disagree, I think the proper hyphenation would be "non-metadata-convention metadata".

Reply:
For some reason "unconventional" always makes me think "unexpected" or "not quite valid" so I have a hard time parsing "unconventional metadata". I agree my comment is verbose but study-specific metadata is allowable metadata, just not stored in the metadata backend.

I've added the hyphenation as you suggest as I agree it was not properly hyphenated before.